test-suite/src/data_type/int128.rs

```diff
@@ -0,0 +1,110 @@
+use {
+    crate::*,
+    gluesql_core::{data::ValueError, prelude::DataType, prelude::Value::*},
+};
+
+test_case!(int128, async move {
+    run!(
+        "CREATE TABLE Item (
+        field_one INT(128),
+        field_two INT(128),
+    );"
+    );
+    run!("INSERT INTO Item VALUES (1, -1), (-2, 2), (3, 3), (-4, -4);");
+
+    let parse_i128 = |text: &str| -> i128 { text.parse().unwrap() };
+
+    let max_str = "170141183460469231731687303715884105728";
+    let min_str = "-170141183460469231731687303715884105729";
+
+    let str = format!("INSERT INTO Item VALUES ({:}, {:})", max_str, max_str);
+
+    test!(Err(ValueError::FailedToParseNumber.into()), &str);
```

Comment:
Though `str` keyword is allowed for variable names, but it overlaps with Rust primitive type `str`.
It would be better to use different names.

In here,
Simply we may not need to use declare variables.
```rust
test!(
    Err(ValueError::FailedToParseNumber.into()),
    &format!("INSERT INTO Item VALUES ({}, {})", i128::MAX, i128::MAX)
);
```
The above would be enough.

However...
It looks that there exists some issues parsing i128 literal
In this test below, it is supposed not to generate error but `FailedToParseNumber` occurs.
```rust
test!(
    Err(ValueError::FailedToParseNumber.into()),
    &format!("INSERT INTO Item VALUES ({}, {})", i128::MAX - 100000, i128::MAX + 100000)
);
```